### ISSUE TITLE
Include deprecated records 2919, 2934, and 3006 in the Syriaca Gazetteer?

### ISSUE THREAD

**Issue by wlpotter** [srophe/bethqatraye-data#27]
https://bqgazetteer.bethmardutho.org/place/3006 - should we keep this place? The abstract indicates uncertainty about what and where this place is.

https://bqgazetteer.bethmardutho.org/place/2972 - this record refers to a settlement, but the description coming from the information from Muqadissi ("Region east of the Caspian Sea.") indicates that Muqadissi is referring to a region. Do we need to split this into two records - one for Jurjan the settlement and another for Jurjan the region?

https://bqgazetteer.bethmardutho.org/place/2952 - The Arabic headword, نهر المنصورة , seems to refer to this place as a river, but the record and abstract indicate a settlement. Is this correct?

**Comment by wlpotter** [comment]
https://bqgazetteer.bethmardutho.org/place/2909 - In the spreadsheet, there are two rows labeled 2909 coming from Ibn Jubayr. One is for the Mosque, but another is for Bab Ibrahim, described as one of the entrances to this Mosque. Do we need a separate record for the Bab Ibrahim?

**Comment by wlpotter** [comment]
https://bqgazetteer.bethmardutho.org/place/2907 - In the spreadsheet, Ibn Battuta has two rows labeled 2907. One is for Medina the settlement. The other is for Baqīʿ al-Ghurqad, "a graveyard in  Medina". Do we need a separate record for the graveyard?

**Comment by wlpotter** [comment]
https://bqgazetteer.bethmardutho.org/place/2919 - The abstract for this record calls it a "water-place". Does this mean an "oasis"? How should the abstract read?

**Comment by wlpotter** [comment]
https://bqgazetteer.bethmardutho.org/place/2934 - The abstract expresses some uncertainty about what this place might be. There is also an unsourced quote in this abstract. Is it perhaps from Ibn Rustah?

**Comment by davidamichelson** [comment]
https://bqgazetteer.bethmardutho.org/place/409 two places combined?

**Comment by wlpotter** [comment]
https://bqgazetteer.bethmardutho.org/place/111 - a note from the Syriaca record mentions that this is different than al-Wasit (place/509). But the BQ placeNames include Wasit. Do we need to separate them?

**Comment by davidamichelson** [comment]
Regarding place http://bqgazetteer.bethmardutho.org/place/2941
We have this as the region of al-Baḥrain but are some of these references to the kingdom of al-Baḥrain? Should we make two entries, one for the kingdom one for the region?

**Comment by wlpotter** [comment]
In place https://bqgazetteer.bethmardutho.org/place/2903.html Ibn Battuta is listed as having both of these names: هجر # الحسا do these refer to the same place?
`<placeName xml:id="name2903-9" xml:lang="ar" source="#bib2903-1">الحسا</placeName>
<bibl xml:id="bib2903-1">
            <title xml:lang="en" level="m">Riḥlat Ibn Baṭṭūta. Tuḥfat al-nuẓẓār fī gharāʾib al-amṣār wa-ʿajāʾib al-asfār</title><author>Ibn Baṭṭūta</author><ptr target="http://syriaca.org/bibl/QVIUC5ZC"/>
            <citedRange unit="pp">287</citedRange>
          </bibl>`

**Comment by wlpotter** [comment]
3006 - delete this record
2972 - split into settlement and region.
2952 - delete nahr (leave name as: المنصورة)
2909 - no record needed for the baab
2907 - no record needed for the graveyard
2919 - delete this record
2934 - delete this record
409 - this is fine
111 - separate out; Kashkar is 111. al-Wasit is 509
2941 - looks to be fine
2903 - wait on this.

**Comment by wlpotter** [comment]
2903 - merge and deprecate 5009 into 2903

**Comment by wlpotter** [comment]
For ones to delete: delete from BQ repo. Decide whether or not they will go into the Syriaca gazetteer

**Comment by wlpotter** [comment]
Remaining items to address:
2972 is settlement Jurjan, create a record (mint URI) for Jurjan the region

111 - Kashkar is 111, remove name references to al-Wasit. Make BQ entry 509 (from Syriaca entry) and add BQ data

2903 - merge and deprecate 5009 into 2903

Decide what to do with 2919, 2934, and 3006 in the Syriaca Gazetteer (include? Delete?)

**Comment by wlpotter** [comment]
5572 was created from 2972. 5572 refers to Jurjan the region. As such, the Wiki and Pleiades URIs were removed from the 'see also' since they appear to refer to the settlement. 

We will need to decide if we keep the coordinates from Pleiades. They are listed as 'representative' so may be okay to keep, but the URI seems to refer to the settlement and not the region.

**Comment by wlpotter** [comment]
All that is left here is to decide what to do with 2919, 2934, and 3006 in the Syriaca Gazetteer (include? Delete?)

**Comment by wlpotter** [comment]
The same issue applies to 5459 (see #43). This was removed from BQG but we may want it in the Syriaca Gazetteer?
